src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/config/DomainEventUrlConfig.kt

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.context.annotation.Configuration
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.DomainEventType
+
+@Configuration
+@ConfigurationProperties(prefix = "url-templates.api")
+class DomainEventUrlConfig {
+  lateinit var cas1: Map<String, String>
+  lateinit var cas2: Map<String, String>
+  lateinit var cas3: Map<String, String>
+
+  fun getUrlForDomainEventType(domainEventType: DomainEventType): String {
+    return when (domainEventType) {
+      DomainEventType.APPROVED_PREMISES_APPLICATION_SUBMITTED -> cas1["application-submitted-event-detail"]
```

Comment:
could we put the template ids into the enum ... this may be overkill :) i guess in hindsight it would have been good to use the enum name to build the template name